template/src/hooks/ColorHook/index.js

```diff
@@ -0,0 +1,10 @@
+import {useGlobalState} from 'tram-one'
+
+const APP_COLOR = 'appColor'
+
+export const useColor = () => {
+    const [colorIndex, setColorIndex] = useGlobalState(APP_COLOR, 0)
+    const colors = ['#e6ef9f', '#a09fef', '#9fefa1', '#ef9f9f']
+    const incrementColor = () => setColorIndex((colorIndex + 1) % colors.length)
+    return [colors[colorIndex], incrementColor]
+}
```

Comment:
Similar to ColorHeader, we should split up the index.js from the source file.

If we could also rename this folder to `useColor`, and make the source file `useColor.js`, that would be appreciated.